utils/position_utils.tsx

```diff
@@ -1,9 +1,13 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-export function popOverOverlayPosition(targetBounds, innerHeight, spaceRequiredOptions) {
+export function popOverOverlayPosition(
+    targetBounds: {top: number; bottom: number},
```

Comment:
You don't need to define the first type in-line, it already exists!
https://developer.mozilla.org/en-US/docs/Web/API/DOMRect

Reply:
That's what I did first but then I thought it was not needed... But sure thing, I can do it !